plugins/headlamp-pod-counter/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "@kinvolk/headlamp-pod-counter",
+  "author": "Kinvolk GmbH <oss@kinvolk.io>",
+  "license": "Apache-2.0",
+  "version": "0.0.1",
+  "description": "Example Headlamp (Kubernetes GUI) plugin. Gets the number of pods in the cluster and displays that information in the top bar.",
+  "scripts": {
+    "start": "headlamp-plugin watch",
+    "build": "headlamp-plugin build"
+  },
+  "keywords": [
+    "headlamp",
+    "kinvolk",
+    "kubernetes",
+    "plugins"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/kinvolk/headlamp.git",
+    "directory": "plugins/headlamp-plugin-counter"
+  },
+  "devDependencies": {
+    "@kinvolk/headlamp-plugin": "file:../headlamp-plugin"
```

Comment:
This needs updating before merging to @kinvolk/headlamp-plugin 0.2.0 once the package has been published.